llbuild.xcodeproj/project.pbxproj

```diff
@@ -2628,12 +2685,31 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				40F638ED205306AB00A1CFBE /* Version.xcconfig in Resources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		40513F1B2057626A00DCA088 /* Generate version.h */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 12;
+			files = (
+			);
+			inputPaths = (
+				"",
```

Comment:
I think there is a stray empty path here...

Reply:
huh, interesting

Reply:
Indeed there was.